### PR TITLE
feat: install media pack on windows with `npx playwright install-deps`

### PR DESCRIPTION
**Pull request by aslushnikov**
Chromium on Windows requires Media Pack to be installed. This patch
moves media pack installation under the `npx playwright install-deps`
umbrella.

### CODE REVIEW

File: bin/install_media_pack.ps1 [hunk 1]
```diff
@@ -0,0 +1 @@
+Install-WindowsFeature Server-Media-Foundation
```
Comment (yury-s) [inline bin/install_media_pack.ps1:1]: Does it work on regular win 10 (non server)?

File: src/install/installDeps.ts [hunk 2]
```diff
@@ -16,15 +16,27 @@
...
 export async function installDeps(browserTypes: string[]) {
-  if (os.platform() !== 'linux')
-    return;
   if (!browserTypes.length)
     browserTypes = ['chromium', 'firefox', 'webkit'];
+  if (os.platform() === 'win32') {
+    if (browserTypes.includes('chromium')) {
+      const {code} = await utils.spawnAsync('powershell.exe', [path.join(SCRIPTS_DIRECTORY, 'install_media_pack.ps1')], { cwd: SCRIPTS_DIRECTORY, stdio: 'inherit' });
+      if (code !== 0)
+        throw new Error('Failed to install windows dependencies!');
```
Comment (yury-s) [inline src/install/installDeps.ts:34]: In the context of the comment above, maybe convert this in a warning?

**Review by yury-s — APPROVED** [review]

File: bin/install_media_pack.ps1 (hunk 1 above)
Comment (aslushnikov) [reply bin/install_media_pack.ps1:1]: Should work! At least according to stack overflow

File: src/install/installDeps.ts (hunk 2 above)
Comment (aslushnikov) [reply src/install/installDeps.ts:34]: I'd keep it failing - it'll be more visible in the CI logs.

**Pull request merged**